apps/core/lib/core/models/user.ex

```diff
@@ -46,9 +46,22 @@ defmodule Core.User do
     |> validate_format(:email, @valid_email_format)
     |> unique_constraint(:email)
     |> validate_length(:password, @valid_password_length)
+    |> validate_confirmation(:password, message: "does not match password!")
     |> encrypt_passwd()
   end
 
+  def update_changeset(model, params) do
```

Comment:
I think I should do it somehow, because in the case of adding a user the password is required, but to update the user is optional.